pyramid_okta/security/authorization.py

```diff
@@ -0,0 +1,5 @@
+__author__ = 'psuresh'
+from pyramid.authorization import ACLAuthorizationPolicy
+
+# Use the default ACL policy and define permissions as ACL's`
+OktaAuthorizationPolicy = ACLAuthorizationPolicy
```

Comment:
not sure if it makes a difference, or which way is better, but could also do:

    class OktaAuthorizationPolicy(ACLAuthorizationPolicy):
        pass